patches.yaml

```
@@ -3972,6 +3972,24 @@ F300_3: # Fire node
     layer: 0
     room: 0
     objtype: OBJ
+  - name: Add Path Idx and SceneFlag to Fire Node Box
+    type: objpatch
+    objtype: OBJ
+    id: 0xFC07
+    layer: 0
+    room: 0
+    object:
+      pathIdx: 1
+      goalscenefid: 48
```

```suggestion
      goalscenefid: 127
```
That's the sceneflag of the fi trigger after tumbleweed chest, we delete the trigger so this sceneflag should be unused